views/layouts/default.lavender

```diff
@@ -52,33 +52,57 @@ body
           i.fa.fa-bars.fa-2x
       .navbar-collapse.collapse
         ul.nav.navbar-nav
-          li(class='#{active_class_index}')
-            a(href='/index.html', title='Home') Home
-          li
-            a(
-              href='https://github.com/golavender/lavender', 
-              title='GitHub', target='_blank'
-              ) GitHub
-          li
-            a(href="http://daftlabs.com", title="daft labs", target="_blank")
-              | Creators
-          li(class='#{active_class_guide} dropdown')
-            a.dropdown-toggle(href='#', data-toggle='dropdown')
-              | Guide
-              span.caret
-            ul.dropdown-menu(role='menu')
-              li
-                a(href='/guide.html') Guide Home
-              li.dropdown-header Installation
-              li
-                a(href='/guide.html#composer', title='GitHub') via Composer
-              li
-                a(href='/guide.html#github') via GitHub
-              li
-                a(href='/guide.html#usage') Usage
-              li.divider
-              li
-                a(href='/guide.html#langref') Language Reference
-          li(class='#{active_class_about}')
-            a(href='/about.html') About
+          -titles = [
+            'Home', 
+            'Github',
+            'Creators', 
+            'Guide', 
+            'Download'
+            'About'
+          ]
+          -url_page = [
+            '/index.html', 
+            'https://github.com/golavender/lavender',
+            'http://daftlabs.com',
+            '/guide.html',
+            '/download.html',
+            '/about.html'
+          ]
+          -target = [
+            '',
+            '_blank',
+            '_blank',
+            '',
+            '',
+            ''
+          ]
```

Comment:
an example of how you can use objects.  pages variable will be an array of objects with properties.  You can access those properties with dot syntax (below)
-pages = [
  {
    title: 'Home',
    url: '/index.html',
    target: '_self'
  },
  {
    title: 'Github',
    url: 'https://github.com',
    target: '_blank'
  },
  (and so on...)
]

each page in pages
  if(current_page == page.title)
    -class_name = 'disabled'
  else
    -class_name = ''
  li(class=class_name)
    a(
      href=page.url
      title=page.title
      target=page.target
    )=page.title